Add new section with instructions for managed services providers to p… (#21209)
DataDog/documentation: content/en/partners/laying-the-groundwork.md

### Single-organization or multiple-organization

> Datadog offers the possibility of managing multiple child organizations from one parent organization. This is the typical deployment model used by MSPs to prevent clients from having access to each others' data. In a multi-org setup, one child organization is created for each client, and the client is restricted to their own child organization. See **Client org provisioning options** below for more information.

Reviewer: ```suggestion
Datadog offers the possibility of managing multiple child organizations from one parent organization. This is the typical deployment model used by MSPs to prevent clients from having access to each others' data. In a multi-org setup, one child organization is created for each client, and the client is restricted to their own child organization. See [Client org provisioning options](#client-org-provisioning-options) for more information.
```

A common question from MSP partners is whether separate Datadog orgs should be set up to manage development, test, and production resources in environments.

Datadog does not recommend separating development, test, and production resources. The recommended approach is to manage all resources in the same Datadog organization and delineate the environments through tags. For more information, see [Tagging strategy][20].

## Client org provisioning options

> If you are managing your clients' Datadog org(s), you may want to control the org provisioning process. Doing so allows you to carry out administrative functions in the org (such as provisioning new users, setting up access methods, defining role-based access, and managing client usage).

Reviewer: ```suggestion
If you are managing your clients' Datadog org(s), you may want to control the org provisioning process and carry out administrative functions in the org such as provisioning new users, setting up access methods, defining role-based access, and managing client usage.

To do so:
```

> 1. Create a child organization under your parent account
> 2. Retrieve the new child organization's OrgID
> 3. Separate the new child organization from your parent account
> 4. Register the new client details in the DPN portal
> 5. Create a child organization under the organization created in step 1 above

Reviewer: ```suggestion
1. Create a child organization under your parent account.
2. Retrieve the new child organization's Org ID.
3. Separate the new child organization from your parent account.
4. Register the new client details in the DPN portal.
5. Create a child organization under the organization created in Step 1 above.
```

> Following this procedure produces the following outcomes:

Reviewer: ```suggestion
As a result:
```

> - a new parent organization is created for the purpose of managing one or more child organizations for your new client
> - the new parent organization and client child organization is registered and attached to a billing contract
> - you are able to provision new users, configure their access methods, define role-based access, and manage the usage for your new client child organization

Reviewer: ```suggestion
- A new parent organization is created for the purpose of managing one or more child organizations for your new client.
- The new parent organization and client child organization is registered and attached to a billing contract.
- You can provision new users, configure their access methods, define role-based access, and manage the usage for your new client child organization.
```

### Create a child organization under your parent account

There are two options for this step:

- Use the UI: Click "New Organization" as described in [Managing Multiple Organization Accounts][1].
- Use the API: Use the [Create a child organization][18] endpoint.

> ### Retrieve the new client org's OrgID

Reviewer: ```suggestion
### Retrieve the new client org's org ID
```

> You can retrieve the OrgId of the Datadog org you are currently logged into by opening the browser's Javascript console and typing:

Reviewer: ```suggestion
You can retrieve the ID of the Datadog org you are logged into by opening the browser's JavaScript console and typing:
```

  `JSON.parse(document.querySelector('#_current_user_json').value).org.id`

> You can also create a bookmark named Get Datadog OrgId whose content is the following Javascript function:

Reviewer: ```suggestion
You can also create a bookmark named `Get Datadog OrgId` which contains the following JavaScript function:
```

  ```javascript:(function() {var orgId = JSON.parse(document.querySelector('#_current_user_json').value).org.id; alert("Datadog OrgId is " + orgId);})();```

> Then, when you are on a Datadog page, click on the bookmark to display the current OrgId in a browser Alert Box.

Reviewer: ```suggestion
Then, when you are on a Datadog page, click on the bookmark to display the current org ID in a browser Alert Box.
```

### Separate the new child organization from your parent account

There are two options for this step:
    - Self-service: Use the [Spin-off Child Organization][19] API endpoint to make the new organization a standalone parent org.
    - Assisted: Contact your Partner Sales Manager to remove the new organization from your parent account.

### Register the new client details in the DPN portal

> - Log in to the [DPN portal][16] and in the click `+Register Deal` on the Deal Dashboard.

Reviewer: ```suggestion
- Log in to the [DPN portal][16] and click `+Register Deal` on the Deal Dashboard.
```

> - Enter the new client details including the new client organization OrgID to register the new client org.

Reviewer: ```suggestion
- Enter the new client details including the new client organization's Org ID to register the new client org.
```

> ### Create a new child organization under the organization created in step 1 above

Reviewer: ```suggestion
### Create a new child organization under the organization created in Step 1 above
```

> 1. Switch to the org created in step 1 above
> 2. Create a client child organization following the instructions in step 1 above

Reviewer: ```suggestion
1. Switch to the org created in Step 1 above.
2. Create a client child organization following the instructions in Step 1 above.
```
